src/views/trade_history_view.js

```diff
@@ -0,0 +1,39 @@
+import Backbone from 'backbone';
+
+const TradeHistoryView = Backbone.View.extend({
+  initialize(params) {
+    // this.trades = []; // took this out because don't need to store this data anymore
+    this.template = params.template;
+
+    // this.listenTo(this.model, 'update', this.bind);
```

Comment:
I think there's a slight change that could be made to sort of "combine" this line and the `bind()` function below, to achieve the same result in a simpler way:
```javascript
this.listenTo(this.model, 'trade', this.render);
```

The trick here is that while `this.model` is referring to the `QuoteList` collection, the way that Backbone works is that if any model inside of a collection triggers an event it "bubbles up" through the collection. So you can listen to the collection directly for any model-specific events, such as `trade`.

A secondary benefit you get from this approach is that you don't need to call `bind()` again if the collection of quotes is expanded.